File: javascript/upload-fragment-cancel.js
var data = "{ }";

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("DELETE", "https://tenant-my.sharepoint.com/alkjl1kjklna");
xhr.setRequestHeader("content-type", "application/json");

xhr.send(data);

File: python/upload-fragment-cancel.py
import http.client

conn = http.client.HTTPSConnection("tenant-my.sharepoint.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("DELETE", "/alkjl1kjklna", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
